src/Umbraco.Infrastructure/Runtime/CoreInitialComposer.cs

```diff
@@ -177,6 +196,170 @@ public override void Compose(Composition composition)
 
             // Config manipulator
             composition.RegisterUnique<IConfigManipulator, JsonConfigManipulator>();
+
+
+            // register the http context and umbraco context accessors
```

Comment:
Should all this 'web' stuff be put into a composition extension like we have for ComposeRespositories, etc.. ?

Reply:
I'm not sure exactly where to do the cut between "Web" and non web. Feel free to give it a shot, but don't see it as an important thing to do